x-pack/plugins/cases/server/services/alerts/index.ts

```diff
@@ -183,7 +183,7 @@ export class AlertService {
     try {
       const docs = alertsInfo
         .filter((alert) => !AlertService.isEmptyAlert(alert))
-        .slice(0, MAX_ALERTS_PER_SUB_CASE)
+        .slice(0, MAX_ALERTS_PER_CASE)
```

Comment:
Do you think we should keep it? It is used mainly by when [pushing](https://github.com/elastic/kibana/blob/3c6e36c25e895058b675831d5069d16440c9a868/x-pack/plugins/cases/server/client/cases/push.ts#L111).

Reply:
Yeah, why don't we leave it in and then when we complete the bulk add for alerts we can refine it.